Public/Src/Engine/Processes/FileAccessPolicy.cs

```diff
@@ -81,6 +81,19 @@ public enum FileAccessPolicy : ushort
         /// </remarks>
         AllowRealInputTimestamps = 0x200,
 
+        /// <summary>
+        /// Override writes allowed by policy based on file existence checks. 
+        /// </summary>
+        /// <remarks>
+        /// Override writes allowed by policy based on file existence checks. 
+        /// Used mainly in the context of shared opaques, where the whole cone under the opaque root is write-allowed by policy (except known inputs).

```

Comment:
Is it not used `entirely` int eh context of shared opaques?
```suggestion
        /// Used entirely in the context of shared opaques, where the whole cone under the opaque root is write-allowed by policy (except known inputs).
```

Reply:
Yeah, guess I was thinking about how this can evolve in the future. WIll change.

---
In reply to: [299145360](https://github.com/microsoft/BuildXL/pull/536#discussion_r299145360) [](ancestors = 299145360)